src/components/TodoComponent/TodoComponent.tsx

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
try to avoid linter disabling

Reply:
i decided to rewrite it, but it still doesn't work